apps/web/lib/event.ts

```diff
@@ -10,7 +10,7 @@ type EventNameObjectType = {
 
 export function getEventName(eventNameObj: EventNameObjectType) {
   return eventNameObj.eventName
-    ? eventNameObj.eventName.replace("{USER}", eventNameObj.attendeeName)
+    ? eventNameObj.eventName.replace("{ATTENDEE}", eventNameObj.attendeeName)
```

Comment:
What will happen with all already created bookings? Since there are many bookings with the `USER` value saved, they won't display the name correctly. Let's try to make this retro compatible.

Reply:
Good point I'll update #2898 with that

Reply:
@zomars pinging this again for rereview